base_layer/wallet/src/output_manager_service/storage/sqlite_db/mod.rs

```diff
@@ -525,93 +562,155 @@ impl OutputManagerBackend for OutputManagerSqliteDatabase {
         Ok(())
     }
 
-    fn update_last_validation_timestamp(&self, hash: FixedHash) -> Result<(), OutputManagerStorageError> {
+    fn update_last_validation_timestamps(&self, hashes: Vec<FixedHash>) -> Result<(), OutputManagerStorageError> {
         let start = Instant::now();
         let mut conn = self.database_connection.get_pooled_connection()?;
         let acquire_lock = start.elapsed();
-        let hash = hash.to_vec();
-        diesel::update(outputs::table.filter(outputs::hash.eq(hash)))
-            .set((outputs::last_validation_timestamp
-                .eq::<Option<NaiveDateTime>>(NaiveDateTime::from_timestamp_opt(Utc::now().timestamp(), 0)),))
+
+        diesel::update(outputs::table.filter(outputs::hash.eq_any(hashes.iter().map(|hash| hash.to_vec()))))
+            .set(outputs::last_validation_timestamp.eq(Some(Utc::now().naive_utc())))
```

Comment:
Perhaps it's best to not update this if the `last_validation_timestamp` is newer than the current timestamp

Reply:
I do not follow. How could `last_validation_timestamp` be newer than the current timestamp?